low-level/framework/utils/storage_encl.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import time
+import json
+
+from framework.base import sspl_constants as sspl_const
+from framework.platforms.realstor.realstor_enclosure \
+    import singleton_realstorencl
+
+
+class StorageEncl:
+    """Base class for storage enclosure related utility functions."""
+    def get_enclosure_logs(self, file_name, logger):
```

Comment:
actually these two functions can easily stay in existing class realstor_enclosure.py, as implementation is realstor specific.
Always look for any existing class to add too. If none, they add new classes. 
I had given this storage_encl class input as an example, to be applied or not applied as needed

Reply:
Actually, I looked into utils for any enclosure related class but there were none, that's why I created new class. 
I have moved the functions to realstor_enclosure.py

Reply:
Done